src/app/organizations/manage/policy-edit.component.ts

```diff
@@ -60,6 +60,9 @@ export class PolicyEditComponent implements OnInit {
     passGenCapitalize?: boolean;
     passGenIncludeNumber?: boolean;
 
+    // Send options
+    sendDisableHideEmail?: boolean;
+
```

Comment:
@cscharf mentioned to remove the line break between this comment and the options, but I see that that is not the style we are using for the other sections here (though I would say it should be). Can we sync these up and remove the other extra lines while in the area?

Reply:
I agree that removing the whitespace is more consistent with our code style. Done